core-data/test/selectors.js

```diff
@@ -56,3 +56,19 @@ describe( 'isRequestingTerms()', () => {
 		expect( result ).toBe( true );
 	} );
 } );
+
+describe( 'getMedia', () => {
+	it( 'returns a media element by id', () => {
+		let state = deepFreeze( {
+			media: {},
+		} );
+		expect( getMedia( state, 1 ) ).toBe( undefined );
```

Comment:
Minor: Seems like a separate test case to me (`it should return undefined for unknown media`)